app/bundles/FormBundle/Translations/en_US/messages.ini

```
@@ -101,7 +101,7 @@ mautic.form.field.form.saveresult="Save result?"
 mautic.form.field.form.showheader="Show header?"
 mautic.form.field.form.showlabel="Show label?"
 mautic.form.field.form.validationmsg="Validation message"
-mautic.form.field.help.auto_fill = "Enable auto fill this value with that of the tracked contact profile's value. If several contacts share the same IP address, incorrect details may be prepopulated. Only works for forms embedded into a Mautic landing page."
+mautic.form.field.help.auto_fill = "Enable auto-fill this value with that of the tracked contact profile's value. If several contacts share the same IP address, incorrect details may be prepopulated. Only works for forms embedded into a Mautic landing page."
```

```suggestion
mautic.form.field.help.auto_fill = "Allow Mautic to auto-fill this value with that of the tracked contact profile's value. If several contacts share the same IP address, incorrect details may be prepopulated. Only works for forms embedded into a Mautic landing page."
```

I think this reads better if we are going to fix the grammar!